src/components/components/audio-player/audio-player.stories.js

```diff
@@ -1,8 +1,16 @@
-import audioPlayer from './audio-player.twig';
+import template from './audio-player.twig';
```

Comment:
Is this a pattern we're going to be following for every component? Just curious/good to know going forward

Reply:
I think it follows their documentation regarding using args https://storybook.js.org/docs/html/writing-stories/introduction#using-args

But it's different than I've done it. Should probably research the benefits and make sure we're doing things the best way